samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -273,7 +273,7 @@ void main(void)
 
 	k_sem_take(&quit_lock, K_FOREVER);
 
-#if defined(CONFIG_NET_IPV6)
+#if defined(CONFIG_NET_IPV4)
```

Comment:
This fix has been mentioned a few times.  I'd like to make the correction in my branch as it's part of review.